### PR TITLE
merged Gotify notifier support MR #76, permission fixes, Gotify bugfix and documentation update

### PR DESCRIPTION
**Pull request by dennisvanderpool**
1. Merged Gotify notifier support MR #76
2. Fixed permission issue when starting up the docker container
3. Updated README.md w.r.t. docker permission issue on config directory
4. Updated README.md w.r.t. gotify support
5. Fixed notifications not sent if only gotify was enabled, bugfix for #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +80,11 @@ docker run \
  marklagendijk/toogoodtogo-watcher
 ```
 
+Note: When using Gotify as notification, make sure to put them on the same network in docker if hosted on the same docker host, e.g.
+docker network create gotify
+and use "--network=gotify" on both containers
+You can than use "http://gotify" on this container if --name gotify is used for the gotify container
```
Comment (YoueG) [inline README.md:86]:
```suggestion
You can then use "http://gotify" on this container if --name gotify is used for the gotify container
```

**Review by benjaminbrgs — APPROVED** [review]

File: config.defaults.json [hunk 2]
```diff
@@ -28,6 +28,12 @@
       "enabled": false,
       "webhookKey": "see README",
       "webhookEvents": ["see README"]
+    },
+    "gotify": {
+      "enabled": false,
+      "url": "http://push.example.de",
```
Comment (marklagendijk) [inline config.defaults.json:34]: Change to example.com

**Comment by marklagendijk** [comment]
How widely used is Gotify?
Normally I like to test things properly before merging, but I don't have the time currently.
Did multiple people already use this?

**Comment by dennisvanderpool** [comment]
@marklagendijk I tested and it works. I switched over to Telegram so I personally won't miss it. @YoueG built it and would like to use it.

File: config.defaults.json (hunk 2 above)
Comment (dennisvanderpool) [reply config.defaults.json]: > Change to example.com
Done

**Pull request merged**